pkg/report/writer.go

```diff
@@ -103,6 +103,15 @@ func Write(report types.Report, option Option) error {
 	return nil
 }
 
+func ValidateFormat(format string) error {
+	switch format {
+	case FormatTable, FormatJSON, FormatGitHub, FormatCycloneDX, FormatSPDX, FormatSPDXJSON, FormatTemplate, FormatSarif, FormatCosignVuln:
+		return nil
+	default:
+		return xerrors.Errorf("unknown format: %v", format)
+	}
```

Comment:
Adding `SupportedFormats` may be better.
https://github.com/aquasecurity/trivy/blob/3daf3dfdf51727a2faff1aad778020e8d7e2826c/pkg/report/writer.go#L35-L37

```suggestion
	return slices.Contains(SupportedFormats, format)
```

Reply:
Also, it would be great to replace these values with `SupportedFormats`.
https://github.com/aquasecurity/trivy/blob/a3eece4fef9294bd1d80e66411a514d3ecc55aa1/pkg/flag/report_flags.go#L29